azurecontainerapps/task.loc.json

```diff
@@ -19,7 +19,7 @@
   "version": {
     "Major": 0,
```

Comment:
Could you please clarify how versioning works on DevOps? If we just upgrade the Patch value, will the existing customers be automatically updated, or will they need a manual action?

Reply:
There's a good snippet [here](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/tasks?view=azure-devops&tabs=yaml#task-versions) that outlines how versioning works for Azure DevOps Task, but the _tldr_ would be that users can _only_ target the major version of a public task and they would get the latest version published for that major version. For example, users target this task via `AzureContainerApps@0`.

When we push out a new task with an updated minor or patch version, customers will consume it automatically as long as they're targeting the major version that the corresponding minor or patch versions were made in.